docs/css/main.css

```diff
@@ -0,0 +1,574 @@
+  font-display: optional;
+}
+/* latin */
+@font-face {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 500;
+  src: local('Poppins Medium'), local('Poppins-Medium'), url(https://fonts.gstatic.com/s/poppins/v5/pxiByp8kv8JHgFVrLGT9Z1xlFd2JQEk.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+  font-display: optional;
+}
+
+body {
+  min-height: 101vh;
+  margin: 0;
+  background-color: #FFF;
+  color: #383838;
+  font-family: 'Roboto', 'Noto', sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 1.5;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  text-rendering: optimizeLegibility;
+}
+
+h1 {
+  font-family: 'Poppins', 'Roboto', 'Noto', sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  letter-spacing: 0.04em;
+}
+
+h2 {
+  font-family: 'Poppins', 'Roboto', 'Noto', sans-serif;
+  font-size: 24px;
+  font-weight: 400;
+  line-height: 1.3;
+}
+
+h3 {
+  font-family: 'Poppins', 'Roboto', 'Noto', sans-serif;
+  font-size: 20px;
+}
+
+h4 {
+  font-family: 'Poppins', 'Roboto', 'Noto', sans-serif;
+  font-size: 16px;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.flex {
+  flex: 1;
+}
+
+/* top nav */
+.main-nav {
+  position: relative;
+  font-family: 'Poppins', 'Roboto', 'Noto', sans-serif;
+  display: flex;
+  align-items: center;
+  padding: 0 48px;
+  box-sizing: border-box;
+  font-size: 16px;
+  letter-spacing: 1px;
+  font-weight: normal;
+  border-bottom: 1px solid #EEEBF3;
+  background: #212124;
+  color: white;
+}
+
+.main-nav > ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.main-nav > ul > li {
+  display: inline-block;
+  padding: 8px 0;
+  margin: 0 12px;
+}
+
+.main-nav > ul > li:hover {
+  color: #FADE4B;
+}
+
+#searchBox {
+  display: inline-block;
+  width: 200px;
+  background: white;
+  border: none;
+  font-family: inherit;
+  font-size: inherit;
+  color: black;
+  margin: 0 10px;
+  border-bottom: 1px solid black;
+  -webkit-appearance: textfield;
+}
+
+#searchBox::-webkit-search-decoration {
+  -webkit-appearance: none;
+}
+
+/* home page */
+.pretty-button {
+  display: inline-block;
+  box-sizing: border-box;
+  margin: 0 4px;
+  padding: 8px 44px;
+  border: 2px solid #000;
+  background-color: transparent;
+  font-size: 14px;
+  font-weight: 500;
+  color: inherit;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  border-radius: 0;
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+.pretty-button:active {
+  background-color: #333;
+  color: #FFF;
+}
+
+.pretty-button:hover {
+  text-decoration: none !important;
+  background-color: #333;
+  color: #FFF;
+}
+
+section {
+  padding: 60px 24px;
+  text-align: center;
```

Comment:
this is causing the 3-column "Why Use lit-html" section in index.html to have centered text. Can we override that or remove the centering?
<img width="893" alt="screen shot 2018-10-05 at 10 16 39 am" src="https://user-images.githubusercontent.com/522948/46549742-dedd8600-c887-11e8-9830-c4a332d4c6c8.png">